### DISCUSSION TITLE
Should following a Lemmy community work?

### DISCUSSION THREAD

**Discussion by davetapley** [pixelfed/pixelfed#4648] [Q&A]
Half philosophical, half technical question here:

Should following a [Lemmy](https://github.com/LemmyNet/lemmy) community work, and if so, how?

---

On https://pixelfed.social/ right now I can:
1. Search e.g. [@dadjokes@lemmy.world](https://lemmy.world/c/dadjokes), it have it show up as **Remote account**
2. Click and [see page for it on pixelfed.social](https://pixelfed.social/i/web/profile/608034257559213695) with some profile populated
3. Follow it
4. No posts yet (will update if one federates over)

---

One immediate quirk: Searching for [!dadjokes@lemmy.world](https://lemmy.world/c/dadjokes) **doesn't work**, and that would _technically_ be the correct syntax to search for a Lemmy community (as opposed to `@` for a user). Mastodon is also dealing with this:
- https://github.com/mastodon/mastodon/issues/17008

But be aware if you test on Mastodon right now:
- https://github.com/mastodon/mastodon/discussions/25623

**Comment by wakest** [comment]
The second issue you mention was fixed a long time ago. Someone with access should mark [#25623](https://github.com/mastodon/mastodon/discussions/25623) solved.

**Comment by wakest** [comment]
Your link number 2 is a link to a locally loaded pixelfed object so you must be logged in for it to load. It actually didn't work even when I was logged in, and only worked once I searched [@dadjokes@lemmy.world](https://lemmy.world/c/dadjokes) first and then it showed up. Then when I click on that link it loads. Sort of strange behavior, not sure what thats about.
